### ISSUE TITLE
Model deployment

### ISSUE THREAD

**Issue by bluesky314** [mapillary/inplace_abn#180]
Is there any provisions to make in production cases when deploying a model with this batch norm on mobile devices or on any server? Can it run smoothly on cpu?

**Comment by ducksoup** [comment]
InPlaceABN models can be deployed by following any standard procedure to deploy Pytorch models using standard BN, with a small caveat: `weight` parameters from all InPlaceABN layers should be replaced with `weight.abs()`. This is necessary because of the different way InPlaceABN and standard BN do feature scaling.
